smart_filter/Makefile

```diff
@@ -1,5 +0,0 @@
-build_test:
```

Comment:
at least some version of this should be kept.

Reply:
I was able to add a `.cargo/config.toml` file with `[build] target = "wasm32-unknown-unknown"` so that cargo build will behave correctly by default, so no need for makefile